tools/fast_puller_.py

```diff
@@ -55,6 +64,11 @@ def main():
   if not args.name or not args.directory:
     raise Exception('--name and --directory are required arguments.')
 
+  if args.cacerts:
+    logging.info('Adding CA certificates of %s', args.cacerts)
+    retry_transport.DEFAULT_SOURCE_TRANSPORT_CALLABLE = _apply_ca_certs(
```

Comment:
This hack shouldn't be necessary. More commentary in https://github.com/google/containerregistry/pull/51

Reply:
Ok, just ping me when you rewrote the retry and I'll change this PR.

Reply:
roger

Reply:
@abergmeier I just upstreamed the internal change. If you rebase, you can supply the RetryTransport factory with your own source transport callable (i.e. a lambda that instantiates a httplib2.Http with the desired certs)